### PR TITLE
DOC Handle email send exceptions

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/silverstripe-framework/issues/10680

### CODE REVIEW

File: en/02_Developer_Guides/10_Email/index.md [hunk 1]
```diff
@@ -261,6 +261,25 @@ $email = Email::create($from, $to, $subject, $body);
...
+$email = Email::create('from@example.com', 'to@example.com', 'My subject');
+$email->text('My plain text email content');
+try {
+    $email->send();
+} catch (TransportExceptionInterface $e) {
+    // handle exception
+}
+```
+
+For more information, refer to [handling sending failures](https://symfony.com/doc/current/mailer.html#handling-sending-failures) in the [symfony/mailer docs](https://symfony.com/doc/current/mailer.html).
```
Comment (ChloeSartorelli) [inline en/02_Developer_Guides/10_Email/index.md]:
```suggestion
For more information, refer to [handling sending failures](https://symfony.com/doc/current/mailer.html#handling-sending-failures) in the symfony/mailer docs.
```
That link is already to the docs. Having two different links to two different parts of the doc for the same thing is just confusing.

**Review by GuySartorelli — CHANGES_REQUESTED** [review]
There's still a `if ($email->send()) {` in one of the examples on this page - change that to use a try/catch instead.

File: en/02_Developer_Guides/10_Email/index.md (hunk 1 above)
Comment (emteknetnz) [reply en/02_Developer_Guides/10_Email/index.md]: Updated

**Comment by emteknetnz** [comment]
I've replaced the `if ($email->send()) ` with simply `$email->send()` without the try/catch block.  The rest of the examples that include `$email->send()` also don't have the try/catch block, so I think it makes sense to only include the try/catch block under the heading to do with catching email send failure exceptions

**Review by GuySartorelli — APPROVED** [review]

**Pull request merged**